src/styles.scss

```diff
@@ -68,9 +68,10 @@ https://github.com/angular/material2/issues/998 */
 }
 
 .badge-notification {
-  margin-top: -46px;
-  margin-left: -15px;
-  background-color: #F44336;
+  margin-top: 15px;
+  position: absolute;
```

Comment:
position:absolute is generally a bad practice, really brittle. Should try to position this without it.